src/custom/components/AccountDetails/index.tsx

```diff
@@ -158,7 +187,16 @@ export default function AccountDetails({
     }
   }, [dispatch, chainId])
   const explorerLabel = chainId && account ? getExplorerLabel(chainId, account, 'address') : undefined
-  const activityTotalCount = (pendingTransactions?.length || 0) + (confirmedTransactions?.length || 0)
+
+  // TODO: for now I'm merging both Pending and Confirmed activities
+  // TODO: if you wan to show them separately just call this hook once with each list
+  const activities = useMultipleActivityDescriptors({
+    chainId,
+    ids: (pendingTransactions || []).concat(confirmedTransactions || []),
+  })
+  // TODO: and also group them separately
+  const activitiesGroupedByDate = groupActivitiesByDay(activities || [])
```

Comment:
Where all the activities are loaded.

As I mentioned in the TODO, pending decision:
Should we group pending and confirmed together or not?